src/tutorials/abilities_advanced.md

```diff
@@ -304,7 +295,7 @@ We're going to add the ability to focus zoom with right click for better aiming!
     2. To protect ourselves from putting this script in the wrong place, we can add an error check into the script. Beneath the above line, add:
 
         ```lua
-        if not WEAPON:IsA('Weapon') then
+        if not WEAPON then
```

Comment:
I agree with these. I think the :IsA() pattern got started before we added :FindAncestorByType(), which either returns the correct type or nil, so this change is good.